### DISCUSSION TITLE
ScaledObject must be deleted before Deployment, but `helm uninstall` doesn't order it that way

### DISCUSSION THREAD

**Discussion by joncrlsn** [kedacore/keda#3183] [Q&A / Need Help] [answered]
When I do a helm uninstall, it deletes the Deployment before the ScaledObject, but it should be the other way around.   This leaves the ScaledObject stranded until someone manually deletes the SO finalizer, and then deletes the SO manually... which isn't an acceptable solution long-term.

### How do others get around this?  

Thank you in advance!

**Comment by tomkerkhove** [comment, marked as answer]
We've recently updated our docs on this - https://keda.sh/docs/2.7/deploy/#uninstall

Not sure what more we can do though :(
